### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by AbdelazizSoliman**
In this pull request :

- [ ] I styled the bookstore like [Zeplin](https://zeplin.io/) design.

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -1,28 +1,26 @@
...
+        {links.map((link) => (
+          <li key={link.text}>
+            <NavLink to={link.path} className={styles.navLink}>
+              {link.text}
+            </NavLink>
+          </li>
+        ))}
+      </ul>
     </div>
+    <IoMdContact className={styles.icon} />
```
Comment (fobadara) [inline src/components/Navbar.js]: [REQUIRED] Kindly ensure the user icon looks just like the one in the template. This trains users to stick customer requirements

image

File: src/components/Book.js [hunk 2]
```diff
@@ -1,69 +1,74 @@
...
-          <p className="progressCompleted">
-            <span className="percentage">
-              {book.completed}
-              %
-            </span>
-            <span className="completedWord">Completed</span>
-          </p>
-        </div>
+      <div className={styles.completed}>
+        <div className={styles.progressCircle} />
```
Comment (fobadara) [inline src/components/Book.js]: [REQUIRED] Kindly ensure that the size of your progress circle is similar to the template.  and it has a greyish colour for the section that does not have a blue background. This is a requirement for this milestone.

image

File: src/components/Book.js
Comment (fobadara) [inline src/components/Book.js:1]: [REQUIRED] It is a requirement for your design to match the Zeplin template. Kindly make the changes pointed out in the design.

image

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @AbdelazizSoliman, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Clean and Dry code
✅ README looks professional

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mwanawabangona — APPROVED** [review]
Hi @AbdelazizSoliman 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested changes!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**